nym-vpn-cli/src/main.rs

```diff
@@ -124,6 +128,22 @@ async fn run_vpn(args: commands::RunArgs) -> Result<()> {
     Ok(())
 }
 
+async fn import_credential(
+    args: commands::ImportCredentialArgs,
+    config_path: Option<PathBuf>,
+) -> Result<()> {
+    let config_path = config_path.expect("config path not set");
+    let raw_credential = match args.credential_type.credential_data {
+        Some(data) => data,
+        None => fs::read(
+            args.credential_type
+                .credential_path
+                .expect("unreachable according to clap"),
```

Comment:
In case we add a third type there, this would be reachable (although caught early). Not sure if there's a better way to handle clap restrictions (maybe something like a named match on `ImportCredentialType` where you can check all possible combinations of the arguments, so that we get compile time checks in case of modifications?)

Reply:
Yes good point, I'll map the clap struct to an enum to catch this